app/components/blog-post-preview.js

```diff
@@ -1,21 +1,21 @@
-import classic from 'ember-classic-decorator';
-import { computed } from '@ember/object';
-import { inject as service } from '@ember/service';
-import Component from '@ember/component';
+import classic from "ember-classic-decorator";
+import { computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Component from "@ember/component";
 
 @classic
 export default class BlogPostPreview extends Component {
   @service
-  i18n;
+  intl;
```

Comment:
This was mostly dead simple -- I just dropped in "intl" everywhere there previously was a "i18n." You can see [here](https://github.com/DockYard/ember-i18n-to-intl-migrator#usage) they want you to use a code-shifting utility, but it was trivial enough for me to just Cmd+F.